util/device_sw_utils/extract_sw_logs.py

```diff
@@ -121,6 +121,8 @@ def get_addr_strings(ro_contents):
         base_addr, size, data = ro_content
         while (str_start < size):
             str_end = data.find(b'\0', str_start)
+            if str_end == -1:
+                break
```

Comment:
Add a comment about why we're skipping this entity (ie: "If its not nul-terminated, it's not a C-string").

Reply:
Done.